### DISCUSSION TITLE
Can I have multiple class fixtures against a test class?

### DISCUSSION THREAD

**Discussion by michaelwiles** [xunit/xunit#2487] [Question] [answered]
Would like do do this:

```
public class FixtureA {}

public class FixtureB {}

public class Test : IClassFixture<A>, IClassFixture<B> {

  [Fact]
  public void TestMethod() {

  }
}
```

So in other words, a test class with multiple test fixtures.

I am under the impression this is not supported? If it isn't why isn't? It _significantly_ reduces the utility of the class fixture concept... 

Personally, I don't think it makes any sense why a test class cannot have multiple class fixtures.

**Comment by bartelink** [comment, marked as answer]
Why not try it? I believe it should work based on [this random test I found with basic searching](https://github.com/xunit/xunit/blob/07663749ab0f62597acc5ff5f163df9f5a0ab8d5/src/xunit.v2.tests/Acceptance/FixtureAcceptanceTests.cs#L237)

**Reply by michaelwiles** [reply]
Thank you. You are right, it does work. There must have been some complexity in my system that stopped it working for some reason.
